### PR TITLE
fix: add raghu-007

### PR DESCRIPTION
**Pull request by raghu-007**
resolves #1697

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/raghu-007.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/50697306/216308165-f770f9df-f77b-430e-a4de-94d0bb4234d3.jpg",
+  "issueId": 66,
```
Comment (Panquesito7) [inline profiles/raghu-007.json:3]: Wrong issue ID.
```suggestion
  "issueId": 1697,
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/raghu-007.json (hunk 1 above)
Comment (raghu-007) [reply profiles/raghu-007.json]: > Wrong issue ID.

Done

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by lewisushindi — APPROVED** [review]

**Comment by raghu-007** [comment]
@schmelto please do check

**Review by schmelto — APPROVED** [review]

**Comment by raghu-007** [comment]
@Panquesito7 pls do review and merge my PR

**Review by priyankarpal — APPROVED** [review]
LGTM 👍🏻 ⚡

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**